bin/lint-readme

```diff
@@ -68,4 +69,10 @@ node_modules/.bin/remark \
 cp README.md README.md.temp
 rewrite <README.md.temp >README.md
 
-node_modules/.bin/eslint -- README.md
+if [[ $module == esm ]] ; then
+  source=module
+else
+  source=script
+fi
+
+node_modules/.bin/eslint --parser-options "sourceType:$source" -- README.md
```

Comment:
For the benefit of future diffs, let's wrap the line above like so:

```bash
node_modules/.bin/eslint \
  --parser-options "sourceType:$source" \
  -- README.md
```

Reply:
:zap: